packages/wdio-cli/tests/launcher.test.ts

```diff
@@ -269,6 +269,17 @@ describe('launcher', () => {
         })
     })
 
+    // describe('formatSpecs', () => {
+    //     it('should return correctly formatted specs', () => {
+    //         const capabilities = { specs: ['/a.js', ['/b.js', '/c.js']] }
+    //         const specFileRetries = 17
+    //         const expected = [{ 'files': ['/a.js'], 'retries': 17 }, { 'files': ['/b.js', '/c.js'], 'retries': 17 }]
+    //         // Expect: [{"files": ["/a.js"], "retries": 17}, {"files": ["/b.js", "/c.js"], "retries": 17}]
+    //         // Get: [{"files": ["./tests/test1.js"], "retries": 17}]
+    //         expect(launcher.formatSpecs(capabilities, specFileRetries)).toStrictEqual(expected)
+    //     })
+    // })
```

Comment:
What happens with this unit test?

Reply:
1. Using Capabilities.RemoteCapability results in this transpile error:
```
packages/wdio-cli/src/launcher.ts(202,41): error TS2345: Argument of type 'RemoteCapabilities' is not assignable to parameter of type 'DesiredCapabilities | W3CCapabilities | MultiRemoteCapabilities'.
  Type '(DesiredCapabilities | W3CCapabilities)[]' is not assignable to type 'DesiredCapabilities | W3CCapabilities | MultiRemoteCapabilities'.
    Type '(DesiredCapabilities | W3CCapabilities)[]' is not assignable to type 'MultiRemoteCapabilities'.
      Index signature is missing in type '(DesiredCapabilities | W3CCapabilities)[]'.
```

2. Stating the obvious, the unit test does nothing, as it is commented.  Being a little more helpful, given the input I would expect this output:
```[{"files": ["/a.js"], "retries": 17}, {"files": ["/b.js", "/c.js"], "retries": 17}]```
but in actual fact get this:
```[{"files": ["./tests/test1.js"], "retries": 17}]```
this is the same issue that I filed in this ticket #6593

Reply:
I commented in the issue. Most of the classes are mocked and don't always reflect original behavior.